modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -34,6 +34,10 @@ function acquia_cms_common_install($is_syncing) {
     $system_theme_config->set('default', 'olivero')->save();
     $system_theme_config->set('admin', "gin")->save();
 
+    // Enable gin_toolbar module.
+    $module_installer = \Drupal::service('module_installer');
```

Comment:
@deepakmishra2 Check line#63 using of module_installer service is duplicate within hook_install

Reply:
done